solana/tests/utils/tbr-wrapper.ts

```diff
@@ -81,99 +81,152 @@ export class TbrWrapper {
     owner: PublicKey;
     feeRecipient: PublicKey;
     admins: PublicKey[];
-  }): Promise<TransactionSignature> {
-    return $.sendAndConfirm(await this.client.initialize(args), this.signer);
+  }): Promise<VersionedTransactionResponse | null> {
+    return $.getTransaction($.sendAndConfirm(await this.client.initialize(args), this.signer));
   }
 
-  async submitOwnerTransferRequest(newOwner: PublicKey): Promise<TransactionSignature> {
-    return $.sendAndConfirm(await this.client.submitOwnerTransferRequest(newOwner), this.signer);
+  async submitOwnerTransferRequest(
+    newOwner: PublicKey,
+  ): Promise<VersionedTransactionResponse | null> {
+    return $.getTransaction(
+      $.sendAndConfirm(await this.client.submitOwnerTransferRequest(newOwner), this.signer),
```

Comment:
All these are still very boiler-plate-y.

I'd replace all instances of `$.getTransaction($.sendAndConfirm(await somePromise, this.signer))` with just `sendConfirmGet(somePromise)`.